src/shared.py

```diff
@@ -192,52 +197,58 @@ def reloadBroadcastSendersForWhichImWatching():
 
 
 def fixPotentiallyInvalidUTF8Data(text):
+    """Sanitise invalid UTF-8 strings"""
     try:
         unicode(text, 'utf-8')
         return text
     except:
         return 'Part of the message is corrupt. The message cannot be' \
-           ' displayed the normal way.\n\n' + repr(text)
+            ' displayed the normal way.\n\n' + repr(text)
 
 
 # Checks sensitive file permissions for inappropriate umask
```

Comment:
you can remove lines 209-211 as you moved it to a docstring.

Reply:
Oh, yes. Thanks.